### DISCUSSION TITLE
Adobe Acrobat install failed:  Error 1602

### DISCUSSION THREAD

**Discussion by radioraheem** [microsoft/winget-cli#1240] [General] [answered]
Attempting to install Adobe Acrobat Reader DC using the latest winget and the error states "Installer failed with exit code: 1602"

1602 appears to be "user cancelled installation" which is definitely not the case.

Tested on the last three releases of winget with the same results.  Tested in a regular cmd prompt and admin level cmd prompt.  Also tested using the --force flag.  No change. 

Any ideas/suggestions?

**Comment by radioraheem** [comment, marked as answer]
This can be ignored/deleted.  Whatever was causing it seems to no longer be an issue as of today.

**Comment by Masamune3210** [comment]
If the issue is no longer present then close the issue please

**Reply by radioraheem** [reply]
My apologies.  I hardly ever use discussions on github so I'm not super familiar with the interface.  I'll close it.
